### ISSUE TITLE
Febrile Neutropenia

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6655]
**Preferred term label:**
Febrile Neutropenia

**Synonyms**
Neutropenic fever

**Definition (free text, please give PubMed ID)**
A single oral temperature of greater than or equal to 101 F, or a temperature greater than or equal to 100.4 F for at least an hour, with an absolute neutrophilic count (ANC) of less than 1500 cells/microliter. 
PMID: 31082146

**Parent term (use hpo.jax.org/app)**
Neutropenia

**Comment by pnrobinson** [comment]
We would code this using a combination of the HPO terms for Fever and Neutropenia.
In general, we try to avoid combinations of findings because this would leave to a combinatorial explosion of terms X+Y for existing terms X and Y.
